### PR TITLE
pointing to career page without nav-bar

### PR DESCRIPTION
**Pull request by jstavh**
idea to replace nav-bar concept yet still point to the career page from the home page

### CODE REVIEW

**Comment by jstavh** [comment]
The first commit has three buttons (Careers, Clients, Contacts) if you want to have a page talking about clients/projects, but the original request was for two links (Careers, Contact Us), so the second commit does this.

File: src/astro/src/components/Front.jsx [hunk 1]
```diff
@@ -15,10 +15,14 @@ const Front = () => {
...
-                    <a href="mailto:hello@granola.team"
-                       className="mb-0 text-2xl sm:text-3xl text-logoOrange z-20"
-                        >hello@granola.team</a></p>
+                <div className="flex flex-row space-x-8 py-8">
+                    <a href="/careers" className="w-fit py-4 px-8 rounded-lg
+                    text-white no-underline text-2md bg-logoOrange">
+                        Careers</a>
+                    <a href="mailto:hello@granola.team" className="w-fit py-4 px-8 
+                    rounded-lg text-white no-underline text-2md bg-logoOrange">
+                        Contacts</a>
```
Comment (robinbb) [inline src/astro/src/components/Front.jsx]: This should be "Contact".

**Review by robinbb — CHANGES_REQUESTED** [review]

**Review by robinbb — APPROVED** [review]

**Comment by robinbb** [comment]
@jstavh I propose bringing the nav bar back, but not for the home page. What do you think?

**Comment by jstavh** [comment]
@robinbb we can do that. I will draft up another PR.

**Pull request merged**